sdk/ml/azure-ai-ml/azure/ai/ml/__init__.py

```diff
@@ -26,12 +26,12 @@
     load_environment,
     load_job,
     load_model,
+    load_model_package,
```

Comment:
This was not manually modified by me, it was automatically modified by the black package, moving line 34 here.

Reply:
![image](https://github.com/Azure/azure-sdk-for-python/assets/23182548/3b659266-72ae-48d1-ab16-11088e6941bf)

Reply:
Looks like in Xingzhi's recent PR, he does not need to change this file. 
Is the isort version difference in your env? 

Find a way to revert this change anyway?

Reply:
I don't think there is a problem with this sorting modification, `load_model_package`  package should be placed on line 29 instead of line 34 in order.